syweb/webclient/app/app.js

```diff
@@ -88,7 +92,7 @@ matrixWebClient.config(['$routeProvider', '$provide', '$httpProvider',
             function ($q, $rootScope) {
             return {
                 responseError: function(rejection) {
-                    if (rejection.status === 403 && "data" in rejection && 
+                    if ("data" in rejection && rejection.data && typeof rejection.data == 'object' &&
```

Comment:
This is cleaning up our 401/403 misnomer. CS API v2 returns 401, so I went with just looking at the response data rather than the status code.